opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -607,6 +607,104 @@ mod tests {
         assert_eq!(data_point.value, 30);
     }
 
+    // "multi_thread" tokio flavor must be used else flush won't
+    // be able to make progress!
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn counter_aggregation_attribute_order() {
```

Comment:
Add a test to ensure that the dedupe + sort is case-sensitive as well.

Reply:
coming next PR